pkg/kube/apisix/apis/config/v2alpha1/types.go

```diff
@@ -99,6 +100,7 @@ type ApisixRouteHTTP struct {
 	Backends       []*ApisixRouteHTTPBackend  `json:"backends" yaml:"backends"`
 	Websocket      bool                       `json:"websocket" yaml:"websocket"`
 	Plugins        []*ApisixRouteHTTPPlugin   `json:"plugins,omitempty" yaml:"plugins,omitempty"`
+	PluginConfigID string                     `json:"plugin_config_id,omitempty" yaml:"plugin_config_id,omitempty"`
```

Comment:
IMHO this is a terrible way to expose the ID of an APISIX configuration in the declarative configuration way, it's not easy for people to configure it.

Another CRD to group a bunch of plugins can be introduced, say `ApisixPluginConfig`, and here we should specify the name of such CR.